openshift-install-powervs

```diff
@@ -1333,6 +1355,17 @@ function process_varfile {
   done
 }
 
+#-------------------------------------------------------------------------
+# Flavor
+#-------------------------------------------------------------------------
+function validate_flavor {
+   [[ -z "$flavortype" ]] &&  error "The -flavor '$flavortype' Please specify an option with flavor."
```

Comment:
If the check is for empty string then no need to display empty variable.

"The -flavor option is not provided" or "The -flavor option is empty"